moveit_ros/planning/collision_plugin_loader/src/collision_plugin_loader.cpp

```diff
@@ -108,7 +108,10 @@ bool CollisionPluginLoader::activate(const std::string& name, const planning_sce
 void CollisionPluginLoader::setupScene(ros::NodeHandle& nh, const planning_scene::PlanningScenePtr& scene)
 {
   if (!scene)
+  {
+    ROS_WARN_NAMED("collision_detection", "Cannot setup scene, PlanningScenePtr is null.");
```

Comment:
Sorry to nitpick this nice PR but it will help keep things tidy in the future.

Can you please change this to:

`static const std::string LOGNAME = "collision_detection";`   <--- Goes near top of file, around L38

`ROS_WARN_NAMED(LOGNAME, "Cannot setup scene, PlanningScenePtr is null.");`

Reply:
We have other files that don't set LOGNAME, so I'd be fine either way